docs/blog/posts/2024/android-0.0.27.md

```diff
@@ -1,17 +1,17 @@
 ---
-title: 'WARP Android release v.0.0.27'
-date: 2024-08-08
+title: 'WARP Android release v.0.0.28'
```

Comment:
The file name is still 27, is that intentional?

Reply:
Ohhh actually no, i was gonna add more things to the next release, good catch!